src/main/kotlin/snyk/common/lsp/SnykLanguageClient.kt

```diff
@@ -172,7 +172,20 @@ class SnykLanguageClient : LanguageClient {
     private fun getSnykCodeResult(project: Project, snykScan: SnykScanParams): Map<SnykCodeFile, List<ScanIssue>> {
         check(snykScan.product == "code") { "Expected Snyk Code scan result" }
         if (snykScan.issues.isNullOrEmpty()) return emptyMap()
-        val map = snykScan.issues
+
+        val includeIgnoredIssues = pluginSettings().ignoredIssuesEnabled
```

Comment:
pluginSettings() should be extracted to a variable

Reply:
Done 🙌